### PR TITLE
Fix all versions in package.json

### PR DESCRIPTION
**Pull request by shans**
I picked the current versions that I have installed locally on the hope those would all work..

There was one exception; I'm currently behind on chromedriver locally and I didn't want to revert to an earlier version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,77 +27,77 @@
...
+    "pouchdb-adapter-memory": "7.1.1",
+    "pouchdb-debug": "7.1.1",
+    "source-map-support": "0.5.16",
+    "sourcemapped-stacktrace": "1.1.11",
     "typescript": "3.7.2",
-    "ws": "~6.0.0"
+    "ws": "6.0.0"
   },
   "repository": {
     "type": "git",
```
Comment (alxrsngrtn) [inline package.json]: Let's fix the versions of our `engines`, too: 
```
"engines": {
    "node": ">= 10.13.0",	 
    "npm": ">= 6.9.0"	 
  }	  
```
We use an .nvmrc to enforce a fixed `node` version. We should do the same or similar for npm.

Comment (shaper) [reply package.json]: Perhaps in a separate PR so we can make sure this goes smoothly before we fiddle that, but defer to @shans thought.

**Comment by shaper** [comment]
The CI build failed with:

```
ERROR: /usr/src/app/particles/Tutorial/Kotlin/5_Collections/BUILD:3:1: error executing shell command: '/bin/bash -c bazel-out/k8-fastbuild/bin/particles/Tutorial/Kotlin/5_Collections/Collections_genrule.sh' failed (Exit 1)
Error linking src/tools/webpack.config.js to build/tools/webpack.config.js EEXIST: file already exists, link 'src/tools/webpack.config.js' -> 'build/tools/webpack.config.js'
```

Seems should not be related to the changes, hmm.

File: package.json [hunk 2]
```diff
@@ -27,77 +27,77 @@
     "watch": "concurrently npm:watch:*"
   },
   "devDependencies": {
```
Comment (alxrsngrtn) [inline package.json]: Shall we take this opportunity to bump dependencies that Github detects as security vulnerabilities: 
I don't have the view in `/arcs`, but this is the view in `/arcs-live`: https://github.com/PolymerLabs/arcs-live/network/alerts

Comment (shans) [reply package.json]: The thought was to freeze the versions as-is and only bump after a release.

I'd hold off on security vulnerability reported updates unless there was a valid threat model and some sense of seriousness.

Comment (sjmiles) [reply package.json]: Sorry I merged before I saw this. In any case, I think we should be stable for a short period before making any more changes.  Also, I think it's important we try out changes on as many systems as we can before we change committed `package.json`.

**Pull request merged**